cv_checker_backend/cv_checker_backend/controllers/job_controller.py

```
@@ -1,18 +1,22 @@
 from cv_checker_backend.db.db_connector import DB_SESSION
+from fastapi import UploadFile, File, HTTPException, Form, Request
 from cv_checker_backend.models.job_models import Job, Cv, CvFeatures
-from cv_checker_backend.controllers.user_controller import get_user_from_session
 from cv_checker_backend.controllers.openai_controller import analyze_cv_by_openai
+from cv_checker_backend.models.user_model import User
 
 
-async def upload_job(session: DB_SESSION, job_title: str = Form(None), job_description: str = Form(...), cvs: List[UploadFile] = File(...), user: dict = Depends(get_user_from_session)):
-    if user["status"] is not STATUS["SUCCESS"]:
-        return user
+async def upload_job(request: Request, session: DB_SESSION, job_title: str = Form(None), job_description: str = Form(...), cvs: List[UploadFile] = File(...)):
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)